### PR TITLE
Fix #2401: Resolve ktlint max line in help test package

### LINKED ISSUE #2401: Resolve ktlint max line in help test package
**Describe the bug**
We are preventing the 100 char length at some places like test case fun name. 
`// ktlint-disable max-line-length`

**Expected behavior**
We need to remove this prevention and fix the function name under 100 char length.

### PR DESCRIPTION
**Pull request by lakshaygarg128**
- Fixes #2401 
  

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/sharedTest/java/org/oppia/android/app/help/HelpFragmentTest.kt [hunk 1]
```diff
@@ -134,7 +134,7 @@ class HelpFragmentTest {
   }
 
   @Test
-  fun openHelpActivity_configurationChanged_scrollRecyclerViewToZeroPosition_showsFAQSuccessfully() { // ktlint-disable max-line-length
+  fun openHelp_configChanged_scrollRecyclerViewToZeroPosition_showsFAQSuccess() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/help/HelpFragmentTest.kt]:
```kotlin
fun testHelpFragment_configChanged_FAQListTitleIsDisplayed()
```

Comment (anandwana001) [reply app/src/sharedTest/java/org/oppia/android/app/help/HelpFragmentTest.kt]: also for the other version of this test

```diff
- fun openHelpActivity_scrollRecyclerViewToZeroPosition_showsFAQSuccessfully()
+ fun testHelpFragment_FAQListTitleIsDisplayed()
```

File: app/src/sharedTest/java/org/oppia/android/app/help/HelpFragmentTest.kt [hunk 2]
```diff
@@ -169,7 +169,7 @@ class HelpFragmentTest {
   }
 
   @Test
-  fun openHelpActivity_openNavigationDrawerAndClose_closingOfNavigationDrawerIsVerifiedSuccessfully() { // ktlint-disable max-line-length
+  fun openHelp_openNavDrawerAndClose_closingOfNavDrawerIsVerifiedSuccess() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/help/HelpFragmentTest.kt]:
```kotlin
fun testHelpFragment_openNavDrawerAndClose_navDrawerIsClosed()
```

**Review by anandwana001 — CHANGES_REQUESTED** [review]
Thanks @lakshaygarg128 

could you also please correct other test name as we don't need `openHelpActivity` as a prefix, we use `testHelpFragment` so correct in a way where the test name is correct towards what actions it perform.

File: app/src/sharedTest/java/org/oppia/android/app/help/HelpFragmentTest.kt (hunk 2 above)
Comment (lakshaygarg128) [reply app/src/sharedTest/java/org/oppia/android/app/help/HelpFragmentTest.kt]: ok

**Comment by lakshaygarg128** [comment]
@anandwana001  Why the first test fail ? pl. help

**Comment by FareesHussain** [comment]
> @anandwana001 Why the first test fail ? pl. help

You can understand just by looking into details
https://github.com/oppia/oppia-android/pull/2418/checks?check_run_id=1660963382

File: app/src/sharedTest/java/org/oppia/android/app/help/HelpFragmentTest.kt [hunk 3]
```diff
@@ -120,7 +120,7 @@ class HelpFragmentTest {
   }
 
   @Test
-  fun openHelpActivity_scrollRecyclerViewToZeroPosition_showsFAQSuccessfully() {
+   fun testHelpFragment_FAQListTitleIsDisplayed() {
```
Comment (FareesHussain) [inline app/src/sharedTest/java/org/oppia/android/app/help/HelpFragmentTest.kt]: Remove a space before the keyword `fun`

**Review by FareesHussain — CHANGES_REQUESTED** [review]
@lakshaygarg128

File: app/src/sharedTest/java/org/oppia/android/app/help/HelpFragmentTest.kt (hunk 3 above)
Comment (lakshaygarg128) [reply app/src/sharedTest/java/org/oppia/android/app/help/HelpFragmentTest.kt]: ok

Comment (lakshaygarg128) [reply app/src/sharedTest/java/org/oppia/android/app/help/HelpFragmentTest.kt]: > > @anandwana001 Why the first test fail ? pl. help
> 
> You can understand just by looking into details
> https://github.com/oppia/oppia-android/pull/2418/checks?check_run_id=1660963382

thanks @FareesHussain

File: app/src/sharedTest/java/org/oppia/android/app/help/HelpFragmentTest.kt [hunk 4]
```diff
@@ -120,7 +120,7 @@ class HelpFragmentTest {
   }
 
   @Test
-  fun openHelpActivity_scrollRecyclerViewToZeroPosition_showsFAQSuccessfully() {
+  fun testHelpFragment_FAQListTitleIsDisplayed() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/help/HelpFragmentTest.kt:123]: testHelpFragment_faqListTitleIsDisplayed

Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/help/HelpFragmentTest.kt]: testHelpFragment_faqListTitleIsDisplayed

File: app/src/sharedTest/java/org/oppia/android/app/help/HelpFragmentTest.kt [hunk 5]
```diff
@@ -134,7 +134,7 @@ class HelpFragmentTest {
   }
 
   @Test
-  fun openHelpActivity_configurationChanged_scrollRecyclerViewToZeroPosition_showsFAQSuccessfully() { // ktlint-disable max-line-length
+  fun testHelpFragment_configChanged_FAQListTitleIsDisplayed() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/help/HelpFragmentTest.kt]: testHelpFragment_configChanged_faqListTitleIsDisplayed

**Review by anandwana001 — CHANGES_REQUESTED** [review]
nit changes

File: app/src/sharedTest/java/org/oppia/android/app/help/HelpFragmentTest.kt (hunk 5 above)
Comment (lakshaygarg128) [reply app/src/sharedTest/java/org/oppia/android/app/help/HelpFragmentTest.kt]: done

File: app/src/sharedTest/java/org/oppia/android/app/help/HelpFragmentTest.kt (hunk 4 above)
Comment (lakshaygarg128) [reply app/src/sharedTest/java/org/oppia/android/app/help/HelpFragmentTest.kt]: done

**Review by anandwana001 — APPROVED** [review]
LGTM, Thanks @lakshaygarg128

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**